packages/terra-action-footer/tests/nightwatch/action-footer-container/ActionFooterContainerTestRoutes.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
lets move these examples out of the nightwatch folder and get rid of it all together.

Reply:
You could put the examples in:
`package
    examples
       test-examples
`
This will set us up to consume terra-dev-site in the future.
Here's an example: https://github.com/cerner/terra-framework/tree/master/packages/terra-aggregator/examples/test-examples

Reply:
Should I move the examples that I have under `packages/terra-site/src/examples/action-footer/` here under `examples/index-examples`?

Reply:
No we will move those later, once we deprecate site. Moving the test examples is an easy win and eliminates mention of nightwatch from the package